### ISSUE TITLE
chmod being reported as unlisted binary

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [webpro/knip#222]
[JoshuaKGoldberg/template-typescript-node-package@`00489b2` (#719)](https://github.com/JoshuaKGoldberg/template-typescript-node-package/pull/719/commits/00489b25d495ba2fd8bfc15b2b67e31b2d6bd73c)

```plaaintext
$ knip

Unlisted binaries (1)
chmod  src/steps/writing/writeStructure.ts
 ELIFECYCLE  Command failed with exit code 1.
```

Should `chmod` always be a recognized built-in?

**Comment by webpro** [comment]
```
❯ last-publish chmod
The last publish date of chmod is 3/7/2014
```

Yes :)

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.21.0. See [Release 2.21.0](https://github.com/webpro/knip/releases/tag/2.21.0) for release notes._

**Comment by webpro** [comment]
Thanks again, Josh!
